### PR TITLE
Mobile Sample Menu Update

### PR DESCRIPTION
**Pull request by singa2000**
[Issue] N/A
[Problem] Some Menu isn't proper working
[Solution] Some Menu is disable

Signed-off-by: cookie <cookie@samsung.com>

### CODE REVIEW

**Review by DohyungLim — COMMENTED** [review]
title menu.. :(

File: examples/mobile/UIComponents/components/buttons/index.html [hunk 1]
```diff
@@ -28,11 +28,11 @@ <h1>
 						Contained Button
 					</a>
 				</li>
-				<li class="ui-li-anchor">
+				<!--<li class="ui-li-anchor">
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/buttons/index.html]: Back up the original file and delete the comments.

File: examples/mobile/UIComponents/components/dialogs/picker.html [hunk 2]
```diff
@@ -23,11 +23,11 @@ <h1>
 						Date picker Dialog (Calendar view)
 					</a>
 				</li>
-				<li class="ui-li-anchor">
+				<!--<li class="ui-li-anchor">
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/dialogs/picker.html]: ditto.

File: examples/mobile/UIComponents/components/etc/graphs/bar-graph.html [hunk 3]
```diff
@@ -25,9 +25,9 @@ <h1>Bar graph</h1>
...
 		</ul>
-        </div><!-- /content -->
-		<script src="../../lib/tau/libs/tau.graph.js" type="text/javascript"></script>
-        <script src="bar-graph.js"></script>
+		</div><!-- /content -->
+		<script src="../../../../lib/tau/libs/tau.graph.js" type="text/javascript"></script>
+		<script src="bar-graph.js"></script>
 </div><!-- /page -->
 </body>
 </html>
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/etc/graphs/bar-graph.html]: add end-line.

File: examples/mobile/UIComponents/components/etc/graphs/dynamic-graph.html [hunk 4]
```diff
@@ -27,9 +27,9 @@ <h1>Graph with changing data</h1>
 		</ul>
         </div><!-- /content -->
         <div class="ui-footer" data-position="fixed">
-                <button id="button">Start</button>
+			<button id="button">Start</button>
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/etc/graphs/dynamic-graph.html:30]: indent check.

File: examples/mobile/UIComponents/components/etc/graphs/static-graph.html [hunk 5]
```diff
@@ -28,7 +28,7 @@ <h1>Graph with static data</h1>
 			</li>
 		</ul>
 		</div><!-- /content -->
-		<script src="../../lib/tau/libs/tau.graph.js" type="text/javascript"></script>
+		<script src="../../../../lib/tau/libs/tau.graph.js" type="text/javascript"></script>
 </div><!-- /page -->
 </body>
 </html>
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/etc/graphs/static-graph.html]: ditto.

File: examples/mobile/UIComponents/components/etc/index.html [hunk 6]
```diff
@@ -59,11 +59,11 @@ <h1>
 						Interactive
 					</a>
 				</li>
-				<li class="ui-li-anchor">
+				<!--<li class="ui-li-anchor">
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/etc/index.html]: ditto.

**Review by DohyungLim — COMMENTED** [review]
please check comments.

**Review by DohyungLim — APPROVED** [review]
LGTM.

**Review by lslachciak — APPROVED** [review]
works well
time/date picker dialogs have some issues but this is not related to this PR

**Pull request merged**